Assets/Scripts/Player/Level generation scripts/CrossGenerationScript.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections;
```

Comment:
Just wondering about the filepath, why is it "Scripts/Player/Level generation scripts" ?
Imo, the path of the level generation scripts should be just "Scripts/LevelGeneration" (no redundency of "scripts" and outside the player folder makes more sense for me)

Reply:
Things got messed up since people moved folders on master while i was working. I am pretty sure i put them in their own folder but i'll double check